front/src/views/sign-up/sign-up.view.js

```diff
@@ -23,7 +23,7 @@ export default {
   methods: {
     async signUp () {
       if (this.valid) {
-        const response = await this.$store.dispatch('signUp', {
+        const response = await this.$store.dispatch('auth/signUp', {
           email: this.email,
           password: this.firstPass
         });
```

Comment:
```
if (response === 'Ok') {
          this.$router.push('/login');
        }
        if (response === 'UnprocessableEntity') {
          this.existingEmail = true;
          this.validate();
          this.existingEmail = false;
        }
      }
```

hm, I was sure it was changed in previous pull request

1. successful codes is automatically resolves and handling could be in then
2. unsuccessful flows should be in catch. it is happened because of it was processed previously in action in catch and then it wasn't rejected, so it became resolved and in next chain went to then
3. comparing with string is really strange - please, leave status or return flags or set of flags